### DISCUSSION TITLE
How to create JBI file with position datas?

### DISCUSSION THREAD

**Discussion by yfc26** [Yaskawa-Global/YMConnect#20] [Show and tell]
Hi, I want to create a JBI file for offline program. How can I add the postion data into the JBI file. And where can I get the meaning of NPOS, RCONF, INST, etc?

**Comment by ted-miller** [comment]
Hello @yfc26 
Please see chapter 4 of the Relative Job Function manual for a description of the job header.
https://www.motoman.com/getmedia/D3D0ED95-7A94-43DA-8A3F-A765F8DB8DFB/178659-1CD.pdf.aspx?ext=.pdf

Position data is stored as `Cxxxxx` tags in the `//POS` section of the header.

![image](https://github.com/user-attachments/assets/c35927b3-0473-4f5a-baea-e5fcb677f7a0)

**Reply by yfc26** [reply]
I understand, thank you very much. Also I want to know will the MotionManagerInterface be available for DX200 in the future?

**Reply by ted-miller** [reply]
No, the MotionManager will not be available for the DX200.
